common/context_mgmt.c

```diff
@@ -111,6 +111,11 @@ static void cm_init_context_common(cpu_context_t *ctx, const entry_point_info_t
 	if (EP_GET_ST(ep->h.attr))
 		scr_el3 |= SCR_ST_BIT;
 
+#ifndef HANDLE_EA_EL3_FIRST
```

Comment:
This new build flag needs documenting in the user guide. Also, we typically allow these flags to be enabled via the build command line by adding them to the makefile and propagating them to the source via the `add_define` make function.